code/drasil-example/dblpend/lib/Drasil/DblPend/Body.hs

```diff
@@ -127,7 +127,7 @@ si = SI {
 }
 
 purp :: Sentence
-purp = foldlSent_ [S "predict the", phrase motion `S.ofA` S "double", phrase pendulum]
+purp = foldlSent_ [S "predict", S "the", phrase motion `S.ofA` S "double", phrase pendulum]
```

Comment:
If you had used the combinator `the`, this would have been good, but this is not.